src/query/QueryManager.js

```diff
@@ -29,18 +29,17 @@ export default class QueryManager {
 
   async querySwisstopo(pickedPosition, layers) {
     const lang = i18next.language;
-    const identifyData = await this.swisstopoIndentify.identify(pickedPosition, layers, lang);
-    if (identifyData) {
-      const {layerBodId, featureId} = identifyData;
-      let popupContent = await this.swisstopoIndentify.getPopupForFeature(layerBodId, featureId, lang);
+    const identifyResult = await this.swisstopoIndentify.identify(pickedPosition, layers, lang);
+    if (identifyResult) {
+      let popupContent = await this.swisstopoIndentify.getPopupForFeature(identifyResult.layerBodId, identifyResult.featureId, lang);
```

Comment:
Why change this?

Reply:
To align with the interface. I can rename the interface or revert the naming here. Just thought it is more descriptive.

Reply:
I meant for `const {layerBodId, featureId} = identifyData;`. You did not like ES6 destructuring syntax? ;)

Reply:
Oh, I think it is way more expressive without sugar.